src/main/java/io/shodo/BasketInformations.java

```diff
@@ -0,0 +1,56 @@
+package io.shodo;
+
+import java.util.HashMap;
+
+/**
+ * Created by thomas on 02/12/2019.
+ * This class is a basket
```

Comment:
This could be more explanatory on what is a basket, their params, and some pointer to documentation